README.md

```diff
@@ -0,0 +1,160 @@
+# TitanicML
+
+Titanic predictions with Machine Learning comparing Logistic Regression, Random Forests and Gradient Boosted Trees from TensorFlow. Including data treatment and exploratory data analysis.
+
+Exploring the Kaggle Titanic Dataset to predict passengers survival.
+
+---
+
+### Summary
+
+1. [EDA](#1-the-notebook-starts-with-exploratory-data-analysis-eda)
+2. [Data Preprocessing](#2-now-it's-time-for-data-preprocessing)
+3. [Logistic Regression](#3-logistic-regression-with-tensorflow)
+4. [Random Forest](#4-random-forest-with-tensorflow)
+5. [Gradient Boosted Trees](#5-gradient-boosted-trees)
+6. [Results and Conclusions](#6-results-and-conclusions)
+
+---
+
+### 1. The notebook starts with Exploratory Data Analysis (EDA) 
+
+In EDA we can visualize the data, found possible errors in it, missing values, look for hints for the better features, and more. 
+
+Data Dictionary *from kaggle*
+
+| Variable | Definition | Key |
+| --- | --- | --- |
+| `survival` |	Survival |	0 = No, 1 = Yes |
+| `pclass` |	Ticket class |	1 = 1st, 2 = 2nd, 3 = 3rd |
+| `sex` |	Sex |	|
+| `Age` |	Age in years | |
+| `sibsp` |	# of siblings / spouses aboard the Titanic | |
+| `parch` |	# of parents / children aboard the Titanic | |
+| `ticket` |	Ticket number | |
+| `fare` |	Passenger fare | |
+| `cabin` |	Cabin number | |
+| `embarked` |	Port of Embarkation | C = Cherbourg, Q = Queenstown, S = Southampton |
+
+#### NaN values:
+- We found out that the column Age has 263 missing values, about 20%. Can be a good feature, we will need to treat this missing values.
+- Cabin column has most of the values missing, more than 75%, difficult to fill with veracity when more than 75% of the values are missing.
+- The other columns don't have serious problems.
+
+#### Separating categorical and numerical columns we can see some more things with .describe():
+- 38,4% of the people survived (not very imbalanced).
+- 843 males vs 466 female (a little imbalanced).
+- SibSp and Parch max of 8 and 9 respectively, which is kind of ok considering the average number of childrens were greater: https://populationeducation.org/wp-content/uploads/2020/04/average-number-children-per-us-family-historic-infographic.pdf
```

Comment:
`SibSp` e `Parch` são termos conhecidos ou comumente usados? Se não, eu usaria outra nomenclatura ou o nome em extenso para evitar a pessoa que ta lendo tenha que voltar na tabela acima pra relembrar o que era essas variáveis.

---
Edit: Ou colocaria nomes mais intuitivos nas variáveis, n precisa fazer isso nesse mas é bom para os próximos projetos.

Reply:
O problema de colocar o significado em parenteses é que teria que ficar colocando varias vezes kk acho que a melhor solucao seria fazer nomes no banco mais significativos. Mas isso pra outros projetos e se for possivel tbm

Reply:
Beleza!